examples/centralized_threshold.rs

```diff
@@ -14,25 +14,25 @@ use std::time::Instant;
 mod aggregate_signature;
 
 const DATA_LENGTH: usize = 48;
-pub(crate) type Data = [u8; DATA_LENGTH];
+use alba::utils::types::Element;
 use digest::{Digest, FixedOutput};
 use sha2::Sha512;
 
 #[derive(Debug, Clone)]
-pub(crate) struct AlbaThresholdSignature<H: Digest + FixedOutput> {
+pub(crate) struct AlbaThresholdSignature<const N: usize, H: Digest + FixedOutput> {
     /// Centralized telescope proof
-    pub(crate) proof: Proof<Data, H>,
```

Comment:
I am not sure to understand why you replace the Data type by a bytearray of size N, Shouldn't this be a proof of `Element<Data>`?